app/controllers/portal/client_logins_controller.rb

```diff
@@ -61,6 +62,9 @@ def check_verification_code
       render :enter_verification_code
     end
 
+    # only adding resetting logic for FYST at this time
+    def unlock_matching_records_for_login!; end
+
```

Comment:
Asked Ryan/Tiffany whether we should do this for GYR but didn't really get a response, so going to leave it for now.

Reply:
ok to implement per ryan!